assets/src/scripts/monitoring-location/components/hydrograph/parameters.js

```diff
@@ -158,7 +158,9 @@ export const plotSeriesSelectTable = function(elem,
                 .attr('scope', 'row');
             paramCdCol.append('span')
                 .text(param => param.description)
-                .call(appendTooltip, param => `Parameter code: ${param.parameterCode}`);
+                .each(function(datum) {
```

Comment:
This bit of code wasn't working because appendTooltip assumes the second parameter is text, not a function. The change below solves the issue while still using the datum to form the title

Reply:
Ooh, that '.each' is neat, I don't remember seeing that used before.